hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch by Cheuk Ting Ho adds support for :pep:`655` `Required` and `NotRequired` as attributes of
```

Comment:
`.rst` needs *two* backticks on each side to be monospaced (whereas markdown only needs one).  Kind of annoying, but the single-backticks here mean something like "try to guess what cross-reference domain this should use, and fall back to italics", which... isn't amazing.  [Preview build here](https://hypothesis--3344.org.readthedocs.build/en/3344/changes.html), via the last item of the "39 successful and 1 skipped checks" list.

Reply:
I never pay attention to that... thanks for pointing that out